docs/source/blog/webhook-blog.md

```diff
@@ -0,0 +1,542 @@
+6. Click **Add Webhook** to save your webhook.
+
+<br/><img src="/images/webhook-blog/webhook-setup.png" alt="Screenshot of the partially configured webhook settings in Label Studio UI." class="gif-border" width="800px" height="732px" />
+
+## Start annotating data in Label Studio
+
+After you set up the project, you can start labeling! 
+
+<br/><img src="/images/webhook-blog/woodpecker-labeled-full.png" alt="Screenshot of the Label Studio labeling UI with a woodpecker sitting on a bird feeder with the beak, head, wing, and body labeled with different colored polygons, and the labeled regions visible on the right hand sidebar." class="gif-border" width="800px" height="527px" />
+
+1. In the Label Studio UI, return to the data manager by clicking **Bird Segmentation** in the breadcrumbs. 
+1. From the data manager for the Bird Segmentation project, click **Label All Tasks**.
+2. Select a label and click points on the image to draw a polygon around parts of the bird. 
+3. To make it easier to draw overlapping polygons, use the eye icon in the **Regions** sidebar to hide polygonal regions after you create them.
+4. When you're finished labeling the parts of the bird, click **Submit** to move onto the next task.
+
+<br/><img src="/images/webhook-blog/owl-labeling.gif" alt="Gif of an adorable owl sitting on pavement being labeled in the Label Studio UI with polygons around its beak, head, wing, and body, hiding the polygons after each is created to avoid accidentally interacting with an already-created polygon." class="gif-border" width="800px" height="" />
+
+Label at least 16 bird images to trigger the SageMaker pipeline.
+
+## What the pipeline does behind the scenes
+
+As you start annotating data, Label Studio saves the annotations to your S3 bucket with the `/annotations/` prefix, and sends an event and payload through the Amazon API Gateway to the AWS Lambda function for each annotation created. 
+
+<br/><img src="/images/webhook-blog/sagemaker-webhooks.png" alt="The same diagram as earlier showing data flow from bird images to an S3 bucket, flowing to Label Studio, which outputs annotations to the S3 bucket and sends a webhook to the Amazon API Gateway, which passes that information to the AWS Lambda function, which triggers the Amazon SageMaker training pipeline when the annotation count reaches at least 16, then outputs the resulting model to the S3 bucket." class="gif-border" width="800px" height="661px" />
+
+After you create at least 16 annotations, the Lambda function invokes the SageMaker pipeline that you created and starts processing the data using the preprocessing script and training the ResNet50 model in the SageMaker pipeline.
+
+To see the status of the pipeline, you need the pipeline execution ARN generated when the Lambda function executes the pipeline. After you retrieve the execution ARN, you can run the following from the command line:
```

Comment:
@KonstantinKorotaev / @farioas any idea how/where they could get the execution arn?